Automattic/pocket-casts-android
# Bump `aboutlibraries` to `11.2.1`

## Description


This PR updates `aboutlibraries` to address https://github.com/Automattic/pocket-casts-android/security/dependabot/19

## Testing Instructions
Install automotive app, open Settings and then Acknowledgements. Verify that a list of libraries is present.

## Screenshots or Screencast 

<img width="1136" alt="image" src="https://github.com/Automattic/pocket-casts-android/assets/5845095/c1ffa1eb-321f-4bea-bfbb-1fde7631bae1">

## Checklist
 
#### I have tested any UI changes...
